components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/util/Utils.java

```diff
@@ -66,6 +69,13 @@
     //This is used to pass the verifyEmail or askPassword claim from preAddUser to postAddUser
     private static ThreadLocal<Claim> emailVerifyTemporaryClaim = new ThreadLocal<>();
 
+    //Error messages that are caused by password pattern violations
+    private static final String[] pwdPatternViolations = new String[]{UserCoreErrorConstants.ErrorMessages
+            .ERROR_CODE_ERROR_DURING_PRE_UPDATE_CREDENTIAL_BY_ADMIN.getCode(), UserCoreErrorConstants.ErrorMessages
+            .ERROR_CODE_ERROR_DURING_PRE_UPDATE_CREDENTIAL.getCode()};
+
+    private static final String PASSWORD_LENGTH_TOKEN = "Password length";
```

Comment:
Use "PasswordJavaRegExViolationErrorMsg" user-store property instead of a static string